src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,113 @@
+library contains the Queue, HashMap, and HashSet classes. The
+[collection package](https://www.dartdocs.org/documentation/collection/latest/collection/collection-library.html)
+contains more specialized classes, such as CanonicalizedMap, MapKeySet, PriorityQueue,
+and support for equality testing.
+
+The Dart SDK libraries are documented at [api.dartlang.org](https://api.dartlang.org/stable/).
+Packages outside of the SDK are published to [pub.dartlang.org](https://pub.dartlang.org/)
+and documented at [dartdocs.org](https://www.dartdocs.org/).
+If you don't find the functionality you need on api.dartlang.org, check dartdocs.org.
+
+Some of the libraries in the SDK have corresponding packages with additional
+functionality. The following table includes some that you should know about:
+
+| **SDK Library** | **Package** |
+| [dart:async](https://api.dartlang.org/stable/dart-async/dart-async-library.html)<br>Support for asynchronous programming with classes such as Future and Stream. | [async](https://www.dartdocs.org/documentation/async/latest/)<br>Utility classes to work with asynchronous computations. For more information, see [Unboxing Packages: async part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html), [part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html), and [part 3](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html). |
+| [dart:collection](https://api.dartlang.org/stable/dart-collection/dart-collection-library.html)<br>Classes and utilities that supplement the collection support in dart:core. | [collection](https://www.dartdocs.org/documentation/collection/latest)<br>Utility functions and classes to make working with collections easier. For more information, see [Unboxing Packages: collection](http://news.dartlang.org/2016/01/unboxing-packages-collection.html). |
+| [dart:convert](https://api.dartlang.org/stable/dart-convert/dart-convert-library.html)<br>Encoders and decoders for converting between different data representations, including JSON and UTF-8.| [convert](https://www.dartdocs.org/documentation/convert/latest/)<br>This package contains encoders and decoders for converting between different data representations.|
+| [dart:core](https://api.dartlang.org/stable/dart-core/dart-core-library.html)<br>Built-in types, collections, and other core functionality for every Dart program. | N/A |
+| dart:js<br>Deprecated. | [js](https://www.dartdocs.org/documentation/js/latest)<br>Methods and annotations to specify interoperability with JavaScript APIs. Use this package instead of dart:js. |
+{:.table .table-striped .nowrap}
+
+## Other useful packages
+
+Developers have written some excellent packages for use by the
+Dart community.  Other popular and useful packages that we recommend,
+in alphabetical order:
+
+[firebase3](https://pub.dartlang.org/packages/firebase3)
+: A Dart wrapper library for the Firebase.
```

Comment:
These descriptions could be a little more helpful. For example, this description should probably link to a description of Firebase, and all the descriptions might benefit from an example of when you'd use them.